### DISCUSSION TITLE
ZBDongle-P VS ZBDongle-E - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#8] [Comments]
# ZBDongle-P VS ZBDongle-E - Haade.fr

Comparison of zigbee sonoff dongle-p and dongle-e keys which SOC to choose for the future of protocol management

https://haade.fr/en/blog/zb-dongle-p-vs-zb-dongle-e-zigbee-sonoff

**Comment by MRezaNasirloo** [comment]
Home assistant's [skyconnect](https://www.home-assistant.io/skyconnect/) uses the same EFR32MG21 chip and they're claiming a future update will bring thread support that will work alongside zigbee at the same time.

**Reply by haade-administrator** [reply]
They don't claim they say they are currently working on a firmware update to make zigbee and Matter work at the same time and that's fine, maybe it's possible depending on the flash and ram memory available on the chip
